manifests/webhook.yaml

```diff
@@ -6,12 +6,21 @@ metadata:
   labels:
     app: admission-webhook
 rules:
-- apiGroups: ["apps"]
-  resources: ["statefulsets"]
-  verbs: ["get"]
-- apiGroups: ["pingcap.com"]
-  resources: ["tidbclusters"]
-  verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["create", "get", "list", "watch", "update"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "create", "update", "delete"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list", "watch","update"]
+  - apiGroups: ["apps","extensions"]
+    resources: ["statefulsets"]
```

Comment:
we don't use stateful sets under extensions group, so IMO `extensions` is not necessary

Reply:
fix